apps/frontend/src/pages/landing/landing.tsx

```diff
@@ -0,0 +1,32 @@
+import { useScrollToHash } from "~/libs/hooks/hooks.js";
+
+import {
+	LandingBackground,
+	LandingCTA,
+	LandingFeatures,
+	LandingFooter,
+	LandingHeader,
+	LandingHero,
+	LandingTestimonial,
+} from "./components/components.js";
+import styles from "./styles.module.css";
+
+const Landing: React.FC = () => {
+	useScrollToHash();
```

Comment:
If we use this hook, do we need to have handleClick logic inside Link component?